server/webapp/WEB-INF/rails.new/app/presenters/api_v1/scms/pluggable_scm_representer.rb

```diff
@@ -16,26 +16,10 @@
 
 module ApiV1
   module Scms
-    class PluggableScmRepresenter < BaseRepresenter
```

Comment:
The convention is 'summary',
Can you rename this to PluggableScmSummaryRepresenter?